webxr/openxr/mod.rs

```diff
@@ -347,6 +367,10 @@ impl OpenXrDevice {
                 Some(_) => {
                     // FIXME: Handle other events
                 }
+                None if stopped => {
+                    // wait a bit before repolling for events
+                    thread::sleep(Duration::from_millis(200));
```

Comment:
Actually I'm not sure this is a good idea, this means the XR thread is indefinitely blocked while we're in the IDLE state, and since the XR thread is the main thread right now (@asajeffrey, can we change this for Hololens?) this means we're blocking the main thread too

Reply:
Yeah, this doesn't look great. I don't know if we can run the HL AR APIs off the main thread, it would be nice if we could.

Reply:
Pretty sure we can, there's no documentation about main thread affinity

Reply:
I specifically asked about thread affinity back when we were starting openxr and was told that openxr can run on any thread.